lib/connections/service.go

```diff
@@ -412,6 +423,14 @@ func (s *Service) connect() {
 	}
 }
 
+func (s *Service) isLANHost(host string) bool {
+	addr, err := net.ResolveIPAddr("ip", host)
```

Comment:
Host here comes from `uri.Host` which includes a port number which, again, probably does not resolve here so everything will be considered non-LAN...